src/test/java/api/requests/scenarios/LoanDueDatesAfterRecallTests.java

```diff
@@ -972,4 +974,53 @@ public void loanDueDateTruncatedOnCheckoutWhenRecallAnywhereInQueue() {
       String loanDueDate = loan.getJson().getString("dueDate");
       assertThat(loanDueDate, is(truncatedLoanDate.toString()));
   }
+  @Test
+  public void loanDateTruncatedWhenRecalledItemItemHasBeenPreviouslyRecalledandRenewed() {
+    JsonObject holds = new JsonObject();
+    holds.put("renewItemsWithRequest", false);
+    holds.put("recallReturnInterval", Period.days(2).asJson());
+    setFallbackPolicies(new LoanPolicyBuilder()
+      .withName("Test")
+      .withHolds(holds)
+      .rolling(Period.days(90))
+      .limitedRenewals(2)
+      .withRecallsMinimumGuaranteedLoanPeriod(Period.days(14))
+      .renewWith(Period.days(30))
+      .renewFromSystemDate());
+
+    final ItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    final IndividualResource response = checkOutFixture.checkOutByBarcode(smallAngryPlanet, rebecca);
+    String loanId = response.getJson().getString("id");
+
+    IndividualResource recall = requestsFixture.place(new RequestBuilder()
+    .recall()
+    .forItem(smallAngryPlanet)
+    .withPickupServicePointId(servicePointsFixture.cd1().getId())
+    .by(usersFixture.james()));
+
+    requestsFixture.cancelRequest(recall);
+
+    loansFixture.renewLoan(smallAngryPlanet, rebecca);
+
+    requestsFixture.place(new RequestBuilder()
+    .recall()
+    .forItem(smallAngryPlanet)
+    .withPickupServicePointId(servicePointsFixture.cd1().getId())
+    .by(usersFixture.charlotte()));
+
+    String expectedDueDate = extractDueDate(DateTime.now(DateTimeZone.UTC).plusDays(14).toString());
```

Comment:
Instead of doing this, the test could freeze the clock in order to allow fixed date and times to be used.